src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -691,10 +738,11 @@ public void onUpdate() {
                     sync(syncPacket, syncToAll);
                 }
             } else {
-
                 ticksInSun = 0;
             }
-
+            if (victim != -1 && player.ticksExisted % 20 == 0) {
```

Comment:
We still need the bite timer.  `ticksExisted` does not start at 0 but at a "random" number if you start feeding. Which results in an inconsistent behavior how long it takes for the first blood transfer.

Probably want to move this block into the `level>0` part above too.

Reply:
fixed as of latest commit.